ioctl/client.go

```diff
@@ -39,8 +39,12 @@ type (
 		Stop(context.Context) error
 		// Config returns the config of the client
 		Config() config.Config
+		// Endpoint returns the address of the member to receive input flag value
+		Endpoint() *string
```

Comment:
if you want to receive input flag value, returning a pointer is not a good way. 
`SetEndpointWithFlag(cb func(...))`, in which `cb` is a callback function, whose value is `StringVar` may be better

Reply:
I will try to modify. Thanks.

Reply:
It's OK, Good idea !  I'v done.